oxd-server/src/test/java/org/xdi/oxd/server/GetTokensByCodeTest.java

```diff
@@ -31,6 +32,7 @@ public void test(String host, String opHost, String redirectUrl, String userId,
         final RegisterSiteResponse site = RegisterSiteTest.registerSite(client, opHost, redirectUrl);
         GetTokensByCodeResponse tokensResponse = tokenByCode(client, site, userId, userSecret, CoreUtils.secureRandomString());
         refreshToken(tokensResponse, client, site.getOxdId());
+        tokenByInvalidCode(client, site, userId, userSecret, CoreUtils.secureRandomString());
```

Comment:
we are creating unit tests here. Lets keep proper names and really keep it as `units`.
So rename current as `whenValidCodeIsUsed_shouldGetTokenInResponse` and then add your unit/case `whenInvalidCodeIsUsed_shouldGet400BadRequest`.